config/vufind/GetThis.yaml

```
@@ -0,0 +1,120 @@
+---
+# This file contains the settings for the GetThis feature
+# See Regex.yaml for comment related to regex
+
+# Either to display the copy number (next to the call number), default false
+showCopyNumber: true
+
+templates:
+  # Elements here correspond to a template file within templates/get-this with .phtml being stripped from the end
```

Possible clarification:

```suggestion
  # Elements here correspond to .phtml template files within templates/record/get-this
  # (e.g. the holdings section configures templates/record/get-this/holdings.phtml)
```